libraries/AP_SmartAudio/AP_SmartAudio.cpp

```diff
@@ -333,14 +350,12 @@ void AP_SmartAudio::send_request(smartaudioFrame_t requestFrame, uint8_t size)
         return;
     }
 
-    uint8_t *request=reinterpret_cast<uint8_t*>(&requestFrame.u8RequestFrame);
-    // pull line low
-    _port->write((uint8_t)0);
+    const uint8_t *request=reinterpret_cast<const uint8_t*>(&requestFrame.u8RequestFrame);
+    //_port->write((uint8_t)0);
```

Comment:
This is the low before send. Reviewing the pr with my sma 2.0 vtx, i needed to switch this on to make it works. If disabled, is not working.

Reply:
ok interesting. I'll try again with mine. It's also possible that we can pull the line low a different way